src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -1524,6 +1526,94 @@ public void searchStudentsInCourse() throws Exception {
         assertThat(tutors).isEmpty();
     }
 
+    public void searchUsersInCourse_searchForAllTutors_shouldReturnAllTutorsAndEditors() throws Exception {
+        var course = database.createCourse();
+        // Test: search for all (no login or name) tutors (tutors includes also editors)
+        var result = searchUsersTest(course, List.of("tutors"), Optional.empty(), numberOfTutors + numberOfEditors, true);
+        assertThat(result.stream().filter(UserPublicInfoDTO::getIsEditor)).hasSize(numberOfEditors);
+        assertThat(result.stream().filter(UserPublicInfoDTO::getIsTeachingAssistant)).hasSize(numberOfTutors);
+    }
+
+    public void searchUsersInCourse_searchForAllInstructor_shouldReturnAllInstructors() throws Exception {
+        var course = database.createCourse();
+        // Test: search for all (no login or name) instructors
+        var result = searchUsersTest(course, List.of("instructors"), Optional.empty(), numberOfInstructors, true);
+        assertThat(result.stream().filter(UserPublicInfoDTO::getIsInstructor)).hasSize(numberOfInstructors);
```

Comment:
```suggestion
       assertThat(result).hasSize(numberOfInstructors).allMatch(UserPublicInfoDTO::getIsInstructor);
```

Reply:
But that is not the same right? I want to filter and then heck the size of the filtered result

Reply:
It's not the same, but from how I understand this test only instructors should get returned, right?